src/components/ProfileForm/profileFormDefs.js

```diff
@@ -11,49 +13,49 @@ const maxLength = 100;
 const usernameRegex = /^[a-z]+([a-z0-9-]+[a-z0-9])?$/i;
 const startsWithNumberRegex = /^[0-9]/i;
 const startsOrEndsWithDash = /(^-|-$)/i;
-const UsernameDef = t.refinement(t.String, (username): boolean => {
+const UsernameDef = tForm.refinement(tForm.String, (username): boolean => {
   return username !== null
     && username.length >= MIN_USERNAME_LENGTH
     && username.length <= MAX_USERNAME_LENGTH
     && usernameRegex.test(username);
 });
 
-UsernameDef.getValidationErrorMessage = (username): string => {
+UsernameDef.getValidationErrorMessage = (username): TranslatedString => {
   if (!usernameRegex.test(username)) {
-    if (startsWithNumberRegex.test(username)) return 'Username can not start with a number';
-    if (startsOrEndsWithDash.test(username)) return 'Username can not start or end with a dash';
-    return 'Only use alpha-numeric characters or dashes.';
+    if (startsWithNumberRegex.test(username)) return t('auth:error.invalidUsername_cantStartWithNumber');
+    if (startsOrEndsWithDash.test(username)) return t('auth:error.invalidUsername_cantStartEndWithDash');
+    return t('auth:error.invalidUsername_useAlphanumericSymbolsOnly');
   }
   if (username.length < MIN_USERNAME_LENGTH) {
-    return `Username should be longer than ${MIN_USERNAME_LENGTH - 1} characters.`;
+    return t('auth:error.invalidUsername_tooShort', { requiredLength: MIN_USERNAME_LENGTH - 1 });
   }
   if (username.length > MAX_USERNAME_LENGTH) {
-    return `Username should be less than ${MAX_USERNAME_LENGTH + 1} characters.`;
+    return t('auth:error.invalidUsername_tooLong', { requiredLength: MAX_USERNAME_LENGTH + 1 });
   }
 
-  return 'Please specify the username.';
+  return t('auth:error.missingData', { missingData: t('auth:formData.username') });
 };
 
-const EmailStructDef = t.refinement(t.String, (email: string = ''): boolean => {
+const EmailStructDef = tForm.refinement(tForm.String, (email: string = ''): boolean => {
   return isValidEmail(email) && email.length <= maxLength;
 });
 
-const PhoneStructDef = t.refinement(t.Object, ({ input }): boolean => {
+const PhoneStructDef = tForm.refinement(tForm.Object, ({ input }): boolean => {
   return isValidPhoneWithoutCountryCode(input);
 });
 
 EmailStructDef.getValidationErrorMessage = (email): string => {
   if (email && !isValidEmail(email)) {
-    return 'Please, check your email address. It may contain only latin letters (a-z), numbers (0-9) and dot (.)';
+    return t('auth:invalidEmailAddress');
   } else if (email && email.length > maxLength) {
-    return `Email should not be longer than ${maxLength} symbols`;
+    return t('auth:invalidEMailAddress_tooLong', { requiredLength: maxLength });
```

Comment:
The mixture of camelCase and snake case bothers me a bit here 😆

Reply:
Aah, that's a mistype :)) will fix that shortly. Sorry for any disturbance that this might caused 😊

Reply:
It's not me it's me OCD 😂